sweep.yaml

```diff
@@ -0,0 +1,23 @@
+program: train.py
+method: bayes
+
+project: information_extraction
+
+metric:
+        name: test_pos_rec
```

Comment:
We should add an F1 metric and use that here. Otherwise, a model could just start to always predict positive labels and get good recall but bad precision (it is odd that on the latest run, precision never dips below 1.0; it's being very cautious about predicting positives)

Reply:
Right, Then I think we should also log F1 score also in train.py file.

Let me update that